x-pack/plugins/observability/public/pages/assets_inventory/assets_inventory.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiDataGrid,
+  EuiDataGridColumn,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiProgress,
+  EuiSpacer,
+} from '@elastic/eui';
+import { Asset } from '@kbn/assetManager-plugin/common/types_api';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import React, { useEffect, useState } from 'react';
+import { usePluginContext } from '../../hooks/use_plugin_context';
+import { ObservabilityPublicPluginsStart } from '../../plugin';
+
+const TestBox: React.FC = ({ children }) => (
+  <div style={{ padding: 20, border: '1px solid magenta' }}>{children}</div>
+);
+
+export function AssetsInventoryPage() {
+  const { ObservabilityPageTemplate } = usePluginContext();
+  const { services } = useKibana<ObservabilityPublicPluginsStart>();
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [hosts, setHosts] = useState<Asset[]>([]);
+
+  useEffect(() => {
+    async function retrieve() {
+      setIsLoading(true);
+      const { hosts } = await services.assetManager.publicAssetsClient.getHosts({ from: 'now-1d' });
```

Comment:
Might be off topic from this PR, but why is the client explicitly named public on the exposed contract? Is it important for the consumer to know that this is the public client? 
I understand from within the plugin why we distinguish them but maybe that should just be an internal thing?

Reply:
Fair question, the fact that there are two clients exposed by the plugin and they take different arguments made me feel like there might be value in distinguishing them, although you could never use the wrong client in the wrong place, so it very much might not matter. Definitely open to renaming.

Reply:
Ah right, there is a server side client and the public (browser) side client. 
It's probably a good idea to give them distinct names since they take different parameters (even if you can't mix them up in code), though the name public really confused me but it's public as in the plugin folder `public`.

Might it make more sense to call it an apiClient? Since it's your client to the Kibaba Assets API, not the assets lib?

Reply:
I think of them both as api clients, so I'm not sure on that one. `browserClient` or `frontEndClient` are probably most accurate, but woof. I may just rename it to generic client, same as the back-end one, for now, and we can clarify it with a better name if/when we come up with one.